package.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=20"
```

Comment:
This could be breaking change. Lets keep node 16 for now, as Playwright is still on node 16.
For development we use node 20 as it has better built-in test runner support.

Reply:
Good catch! I thought this only applied when installing separately

Reply:
done